src/main/java/org/jenkinsci/plugins/mib3flash/Mib3Flash.java

```diff
@@ -0,0 +1,122 @@
+package org.jenkinsci.plugins.mib3flash;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.InputStream;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import hudson.Extension;
+import hudson.model.ParameterDefinition;
+import hudson.model.ParameterValue;
+import hudson.model.StringParameterValue;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.StaplerRequest;
+
+public class Mib3Flash extends ParameterDefinition {
+    private String login = null;
+    private String host = null;
+    private String command = null;
+
+    @DataBoundConstructor
+    public Mib3Flash(String name, String description, String login, String host, String command) {
+        super(name, description);
+
+        this.login = login;
+        this.host = host;
+        this.command = command;
+    }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public String getHost() {
+        return host;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    @Override
+    public ParameterValue createValue(StaplerRequest req, JSONObject jo) {
+        StringParameterValue value = req.bindJSON(StringParameterValue.class, jo);
+        value.setDescription(getDescription());
+        return value;
+    }
+
+    @Override
+    public ParameterValue createValue(StaplerRequest req) {
+        return null;
+    }
+
+    public Map<String, String> getEntries() {
```

Comment:
Czy na pewno twoją intencją jest `getEntries`? czy nazwa tej metody nie powinna być bardziej związana z tym co zwracasz w rzeczywistości. np `getMib3BoardFlashes` czy  coś w tym stylu. Jeśli zmienisz tu to pamiętaj o odpowiedniej konwencji w pliku jelly.